packages/server/postgres/migrations/1635354572171_addIntegrationProviderAndToken.ts

```diff
@@ -58,17 +55,13 @@ export async function up() {
     CREATE INDEX IF NOT EXISTS "idx_IntegrationProvider_orgId" ON "IntegrationProvider"("orgId");
     CREATE INDEX IF NOT EXISTS "idx_IntegrationProvider_teamId" ON "IntegrationProvider"("teamId");
     CREATE TABLE IF NOT EXISTS "IntegrationToken" (
```

Comment:
Should we rename it to UserIntegration? For example with webhook integration provider there will be no token but we'll still have an entry in this table, so `IntegrationToken` could be misleading

Reply:
you've got a point, i just think of a webhook URL as a token & then I can sleep at night 😅